addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxPlayerHandler.java

```diff
@@ -0,0 +1,432 @@
+	 */
+	private boolean playing;
+
+	/**
+	 * Creates SqueezeBox Player Handler
+	 * 
+	 * @param thing
+	 */
+	public SqueezeBoxPlayerHandler(Thing thing) {
+		super(thing);
+	}
+
+	@Override
+	public void initialize() {
+		mac = getThing().getConfiguration().as(SqueezeBoxPlayerConfig.class).mac;
+		timeCounter();
+		if (squeezeBoxServerHandler() != null) {
+			updateStatus(
+					squeezeBoxServerHandler().getThing().getStatus());
+		}
+	};
+
+	@Override
+	public void dispose() {
+
+		// stop our duration counter
+		if (timeCounterJob != null && !timeCounterJob.isCancelled()) {
+			timeCounterJob.cancel(true);
+			timeCounterJob = null;
+		}
+
+		if (squeezeBoxServerHandler() != null) {
+			squeezeBoxServerHandler().removePlayerCache(mac);
+		}
+		logger.debug("Thing {} disposed.", getThing().getUID());
+		super.dispose();
+	}
+
+	@Override
+	public void handleCommand(ChannelUID channelUID, Command command) {
+		if (squeezeBoxServerHandler() == null) {
+			logger.warn("Player has no server configured, ignoring command");
+			return;
+		}
+		String mac = getConfigAs(SqueezeBoxPlayerConfig.class).mac;
+		switch (channelUID.getIdWithoutGroup()) {
+		case CHANNEL_POWER:
+			if (command.equals(OnOffType.ON)) {
+				squeezeBoxServerHandler().powerOn(mac);
+			} else {
+				squeezeBoxServerHandler().powerOff(mac);
+			}
+			break;
+		case CHANNEL_MUTE:
+			if (command.equals(OnOffType.ON)) {
+				mute();
+			} else {
+				squeezeBoxServerHandler().unMute(mac, unmuteVolume);
+			}
+			break;
+		case CHANNEL_STOP:
+			if (command.equals(OnOffType.ON)) {
+				squeezeBoxServerHandler().stop(mac);
+			} else {
+				squeezeBoxServerHandler().play(mac);
+			}
+			break;
+		case CHANNEL_VOLUME:
+			if (command instanceof PercentType) {
+				squeezeBoxServerHandler().setVolume(mac,
+						((PercentType) command).intValue());
+			} else if (command.equals(IncreaseDecreaseType.INCREASE)
+					|| command.equals(UpDownType.UP)) {
+				squeezeBoxServerHandler().volumeUp(mac, currentVolume());
+			} else if (command.equals(IncreaseDecreaseType.DECREASE)
+					|| command.equals(UpDownType.DOWN)) {
+				squeezeBoxServerHandler().volumeDown(mac, currentVolume());
+			} else if (command.equals(OnOffType.OFF)) {
+				mute();
+			} else if (command.equals(OnOffType.ON)) {
+				squeezeBoxServerHandler().unMute(mac, unmuteVolume);
+			}
+			break;
+		case CHANNEL_CONTROL:
+			if (command instanceof PlayPauseType) {
+				if (command.equals(PlayPauseType.PLAY)) {
+					squeezeBoxServerHandler().play(mac);
+				} else if (command.equals(PlayPauseType.PAUSE)) {
+					squeezeBoxServerHandler().pause(mac);
+				}
+			}
+			if (command instanceof NextPreviousType) {
+				if (command.equals(NextPreviousType.NEXT)) {
+					squeezeBoxServerHandler().next(mac);
+				} else if (command.equals(NextPreviousType.PREVIOUS)) {
+					squeezeBoxServerHandler().prev(mac);
+				}
+			}
+			if (command instanceof RewindFastforwardType) {
+				if (command.equals(RewindFastforwardType.REWIND)) {
```

Comment:
My question was whether the binding should do "one step for one command" and thus the web controls and the physical remotes have to repeatingly send commands (while the button is pressed). As a user, I expect that while I am pressing the button it keeps fast-forwarding, so this much be achieved somehow.